spring/boot2-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaAutoConfigurationWithoutMeterTest.java

```diff
@@ -72,16 +68,10 @@ private String newUrl(String scheme) {
 
     @Test
     public void test() {
-        final AggregatedHttpResponse msg = WebClient.of(newUrl("h1c"))
-                                                    .get("/ok")
-                                                    .aggregate().join();
-        assertThat(msg.status()).isEqualTo(HttpStatus.OK);
-        assertThat(msg.contentUtf8()).isEqualTo("ok");
-
         final String metricReport = WebClient.of(newUrl("http"))
                                              .get("/internal/metrics")
                                              .aggregate().join()
                                              .contentUtf8();
-        assertThat(metricReport).contains("# TYPE armeria_server_response_duration_seconds_max gauge");
+        assertThat(metricReport).contains("# TYPE armeria_server_connections gauge");
```

Comment:
Sometimes, it takes time to collect metrics by Micrometer and expose them.

How about using Awaitility?
```java
await().untilAsserted(() -> {
	final AggregatedHttpResponse msg = WebClient.of(newUrl("h1c"))
    ...
    assertThat(...)
});
```

Reply:
Fixed 😄